src/test/jtx/TrustedPublisherServer.h

```diff
@@ -45,31 +50,118 @@ class TrustedPublisherServer
     using error_code = boost::system::error_code;
 
     socket_type sock_;
+    endpoint_type ep_;
     boost::asio::ip::tcp::acceptor acceptor_;
     std::function<std::string(int)> getList_;
 
-public:
+    // The SSL context is required, and holds certificates
+    bool useSSL_;
+    boost::asio::ssl::context sslCtx_{boost::asio::ssl::context::tlsv12};
```

Comment:
I wonder if it would be a good idea to use a global constexpr so that all connections are synced on the method version.